packages/astro/src/@types/astro.ts

```diff
@@ -1010,6 +1012,71 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		service: ImageServiceConfig;
+
+		/**
+		 * @docs
+		 * @name image.domains (Experimental)
+		 * @type {Array<string>}
+		 * @default `{domains: []}`
+		 * @version 2.9.2
+		 * @description
+		 * Defines a list of domains allowed to be optimized locally
+		 *
+		 * The value should be an array of domain strings of images allowed to be optimized locally.
+		 * Wildcards are not permitted (see image.remotePatterns).
```

Comment:
```suggestion
		 * Defines a list of permitted image source domains for local image optimization. No other remote images will be optimized by Astro.
		 *
		 * This option requires an array of individual domain names as strings. Wildcards are not permitted. Instead, use [`image.remotePatterns`](#imageremotepatterns-experimental) to define a list of allowed source URL patterns.
```
I'd like to see something more in this style maybe? (Make the wording accurate!)